be/src/vec/functions/function_java_udf.h

```diff
@@ -89,9 +89,12 @@ class JavaFunctionCall : public IFunctionBase {
     PreparedFunctionPtr prepare(FunctionContext* context, const Block& sample_block,
                                 const ColumnNumbers& arguments, size_t result) const override {
         return std::make_shared<JavaUdfPreparedFunction>(
-                std::bind<Status>(&JavaFunctionCall::execute_impl, this, std::placeholders::_1,
-                                  std::placeholders::_2, std::placeholders::_3,
-                                  std::placeholders::_4, std::placeholders::_5),
+                [this](auto&& PH1, auto&& PH2, auto&& PH3, auto&& PH4, auto&& PH5) {
```

Comment:
For most situations, we can replace `auto&&` suggested by clang-tidy with concrete type. there's no need to use template here.